src/mlpack/core/util/version.hpp

```diff
@@ -16,9 +16,9 @@
 
 // The version of mlpack.  If this is a git repository, this will be a version
 // with higher number than the most recent release.
-#define MLPACK_VERSION_MAJOR 3
-#define MLPACK_VERSION_MINOR 4
-#define MLPACK_VERSION_PATCH 3
+#define MLPACK_VERSION_MAJOR 4
+#define MLPACK_VERSION_MINOR 0
+#define MLPACK_VERSION_PATCH 1
```

Comment:
should not the last one be 0?

Reply:
this comment should be ingnored

Reply:
Yep, mlpack-bot will use the previous commit to tag the release, which has the patch version as 0. :+1: